File: Ocaml/src/utilisateur.ml
class utilisateur ((login:string), (nom:string),(prenom:string),(mail:string),(pwd:string),(pays:string)) =
object
val login = login
val nom = nom
val prenom = prenom
val mail = mail
val pwd = pwd
val pays = pays
      
method get_utilisateur = 
	Printf.printf "Login: %s\n" login;
	Printf.printf "Nom: %s\n" nom;
	Printf.printf "Prenom: %s\n" prenom;
	Printf.printf "Mail: %s\n" mail;
	Printf.printf "Pwd: %s\n" pwd;
	Printf.printf "Pays: %s\n" pays;

(*
method virtual deconnecter = bool
method connecter = bool
method connection = 0
method connecter=(login,pwd)=bool
method estconnecte= bool
method verifierpwd(pwd,login)= bool
method inscrire(nom,prenom,mail,pwd,pays)= bool
method inscription(nom,prenom,mail,pwd,pays)= bool
method supprimerinfo = 0
method modifierinfo(login) = 0
method supprimerutilisateur(login)= 0
method selectionner = 0
method verifconnecte = 0
method suprimer = 0
method modifier(nom,prenom,mail,pwd,pays)= bool
method majinfos(nom,prenom,mail,pwd,pays)= bool
*)

end ;;

File: TempDomi/Serveur/UTILISATEUR.java
package Serveur;

public class UTILISATEUR {
	public String Login;
	public String Nom;
	public String Prenom;
	public String Mail;
	public String Pwd;
	public String Pays;

	public Attribuer attribuer;

	public void Creer(String Login, String Nom, String Prenom, String Mail, String Pwd, String Pays){
		this.Login = Login;
		this.Nom = Nom;
		this.Prenom = Prenom;
		this.Mail = Mail;
		this.Pwd = Pwd;
		this.Pays = Pays;
	}
	
	public String GetLogin() {
		return Login;
    }
	
	public String GetNom() {
		return Nom;
    } 

	public String GetPrenom() {
		return Prenom;
    } 

	public String GetMail() {
		return Mail;
    } 

	public String GetPwd() {
		return Pwd;
    } 

	public String GetPays() {
		return Pays;
    } 
	
	
	
	
	
	
	
	public boolean Connecter() {        
        // your code here
        return false;
    }
		
	public Boolean Deconnecter() {        
        // your code here
        return null;
    } 

	public void Deconnexion() {        
        // your code here
    } 

	public boolean Connecter(String Login, String Pwd) {        
        // your code here
        return false;
    } 

	public void VérifierPwd(String Pwd, String Login) {        
        // your code here
    } 

	public Boolean Inscrire(String Nom, String Prenom, String Mail, String Pwd, String Pays) {        
        // your code here
        return null;
    } 

	public Boolean Inscription(String Nom, String Prenom, String Mail, String Pwd, java.util.List Pays) {        
        // your code here
        return null;
    } 

	public void SupprimerInfos() {        
        // your code here
    } 

	public void ModifierInfos() {        
        // your code here
    } 

	public void SupprimerUtilisateur(String Login) {        
        // your code here
    } 

	public Boolean SetPermission(int Id_Perm, String Lib_Perm) {        
        // your code here
        return null;
    } 

	public Boolean VérifConnecté() {        
        // your code here
        return null;
    } 

	public void supprimer() {        
        // your code here
    } 
 
	public Boolean EstConnecté() {        
        // your code here
        return null;
    } 

	public void Selectionner() {        
        // your code here
    } 

	public Boolean Modifier(String Nom, String Prenom, String Mail, String Pwd, String Pays) {        
        // your code here
        return null;
    } 

	public Boolean MajInfos(String Nom, String Prenom, String Mail, String Pwd, String Pays) {        
        // your code here
        return null;
    } 

	public void GetPermissions() {        
        // your code here
    } 

	public Boolean ajouterPermission(int Id_Perm) {        
        // your code here
        return null;
    } 

	public Boolean RetirerPermission(int Id_Perm) {        
        // your code here
        return null;
    } 

	public Boolean DefinirPermission(int Id_Perm, String Lib_Perm) {        
        // your code here
        return null;
    } 
}